pages/linux/wdctl.md

```diff
@@ -0,0 +1,17 @@
+# wdctl
+
+> Shoes hardware watchdog status.
+> The Linux kernel can reset the system if serious problems are detected. This can be implemented via special watchdog hardware, or via a slightly less reliable software-only watchdog inside the kernel.
```

Comment:
This doesn't sound important for this command. Can we remove it?

Reply:
```suggestion
```